src/extensions/default/bramble/lib/launcher.js

```diff
@@ -35,6 +35,10 @@ define(function (require, exports, module) {
                 return;
             }
             browser.update(urlOrHTML);
+
+            if(typeof callback === "function") {
```

Comment:
I'm breaking the contract on `launch`, which should be sync and have no callback at all.  Callers from within Brackets code proper won't use it.  I only use it for my own needs elsewhere in our code.